conda_requirements.txt

```diff
@@ -3,3 +3,5 @@ geopy==1.11.0
 humanize==0.5.1
 pandas>=0.18.1
 seaborn==0.7.1
+numpy==1.11.2
```

Comment:
We don't need that. `numpy` is [packaged by default](https://docs.continuum.io/anaconda/pkg-docs) in Anaconda Python — which is [the recommended Python](https://github.com/datasciencebr/serenata-de-amor/blob/master/CONTRIBUTING.md#environment) for this project.